resource-metadata/hypothesis.md

```diff
@@ -0,0 +1,139 @@
+# Resource Metadata Model
+
+## Background
+
+*This section is non-normative*
+
+This document introduces a mechanism for linking to metadata about resources in the Solid Ecosystem using HTTP Link headers. Examples of this linking mechanism include:
+
+- A binary JPEG image with a Link header, pointing to a description that has an RDF representation.
+- An LDP container with a Link header, pointing to an ACL resource that governs access controls for the contained resources.
+- A resource whose shape is constrained by a ShEx document includes a Link header that points to that ShEx resource.
+- A resource may keep an audit trail of modification events. A client could follow the URI in a Link header to access that audit trail.
+
+A given resource might link to zero or more such related metadata representations. Having a unified model to describe how clients and Solid servers interact with and process these resources will help clarify expectations, while also providing a shared pattern for extending the features of Solid server implementations.
+
+The metadata model described in this document makes it possible for clients to decorate resources with structured descriptive data. Those metadata may serve as supplementary descriptions or, when supported by a Solid server implementation, they may influence the behavior of the resources.
+
+Some examples include:
+
+- An ACL resource controls how a Solid server makes authorization decisions for a container and any child resources.
+- A shape constraint resource may limit the RDF structures that can be added to a resource or container.
+- A “configuration” resource may control how a resource is versioned, or which indexes are exposed for it.
+
+Access to different categories of metadata may require different levels of privilege, which must be specified as part of the definition for a given metadata type.
+
+## Metadata Discovery
```

Comment:
To give an overview, I think text and table along the the following lines can help (with some inspiration from the fragment specification table under https://www.w3.org/TR/annotation-model/#model-18 ):

"
The following are some of the resource metadata types and the relation URIs that are used to discover the resource metadata. Other metadata types and relations may also be used.

Type | Relation | Description
--- | --- | ---
Access control | `acl` or `http://www.w3.org/ns/solid/terms#acl` | [[WAC](http://solid.github.io/web-access-control-spec/)]
Resource description  | `describedby` or `https://www.w3.org/ns/iana/link-relations/relation#describedby` | ...
Shape | `http://www.w3.org/ns/solid/terms#shape` | ...
Server managed | `http://www.w3.org/ns/solid/terms#servermanaged` | ...
Configuration | `http://www.w3.org/ns/solid/terms#configuration` | ...

"